### ISSUE TITLE
CommandBar with farItems={[]} results in a11y violation - should treat [] same as undefined?

### ISSUE THREAD

**Issue by BrendanMcK** [OfficeDev/office-ui-fabric-react#11650]
### Environment Information

- **Package version(s)**: 7.79.0
- **Browser and OS versions**: n/a

### Describe the issue:

If CommandBar is passed a value of [] for `farItems`, it ends up rendering an empty DIV that has role=menubar for the 'far' part of the menubar. While this looks fine visually, it results in an a11y violation from axe-core based tools (eg. Accessibility Insights), since menubars must have at least one menuitem or similar child.

Suggest treating [] same as undefined, and not rendering a 'far' div at all if there's no items to render.

We hit this in one of our internal components that wraps CommandBar, and then filters some items: this wrapper had a default value of [] for farItems, so we ended up with this a11y violation. We can fix locally to pass through `undefined` as-is - but for robustness and to benefit other Fabric users, may want to update the underlying control itself.

### If applicable, please provide a codepen repro:

https://codepen.io/brendanmckeon/pen/oNgqxZB

Inspect the DOM - can observe an empty DIV with role=menuitem:

![image](https://user-images.githubusercontent.com/1754373/72036100-a973d500-324e-11ea-8a64-7e429fd35a98.png)

### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No): If time permits...

Requested priority: (Blocking, High, Normal, Low): Normal/Low

Products/sites affected: (if applicable)

**Comment by xugao** [comment]
@BrendanMcK - thank you for reporting! agreed that we should not render empty div there. i will fix
